### PR TITLE
Add task-2 script

### PR DESCRIPTION
**Pull request by sggts04**
__2__ 

#### Short description of what this resolves:
task2.py is the script to organise the files.

### CODE REVIEW

**Comment by sggts04** [comment]
@Lokesh2703 please review

**Comment by sggts04** [comment]
@deepakDOTexe please review

**Comment by Lokesh2703** [comment]
**@sggts04** Use os.path.join to go to directory and "\\" is windows specific joiner.

**Comment by sggts04** [comment]
@Lokesh2703 fixed

File: Submissions/sggts04/Task_2/task2.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import os
+import shutil
+
+os.chdir(os.path.join(os.getcwd(), "My_Files"))
```
Comment (Lokesh2703) [inline Submissions/sggts04/Task_2/task2.py]: Try to refer the folder which we gave instead of copying My_Files in this folder

Comment (Lokesh2703) [reply Submissions/sggts04/Task_2/task2.py]: You can use for example os.chdir('../../../ur_folder')

Comment (sggts04) [reply Submissions/sggts04/Task_2/task2.py]: Fixed now

**Pull request merged**